webapp/channels/src/components/post_view/post_time/post_time.tsx

```diff
@@ -105,6 +105,7 @@ export default class PostTime extends React.PureComponent<Props> {
                             value={eventTime}
                             ranges={POST_TOOLTIP_RANGES}
                             useSemanticOutput={false}
+                            displayTimeWithSeconds={true}
```

Comment:
```suggestion
                            useTime={getTimeFormat}
```

```tsx
// static at the top or somewhere
const  getTimeFormat: ComponentProps<typeof Timestamp>['useTime'] = (_, {hour, minute, second}) => ({hour, minute, second});

Reply:
This will show timestamps with seconds for all scenarios, but the issue says it should only be for the tooltip, and as it is a shared component that is why I introduced a flag.

Reply:
This is the tooltip timestamp. There's a separate timestamp declared up on line 67 for the non-tooltip part.

Reply:
Got it.